container_service_extension/config_validator.py

```diff
@@ -103,8 +105,11 @@ def get_validated_config(config_file_name,
     msg_update_callback.info(
         f"Validating config file '{config_file_name}'")
     # This allows us to compare top-level config keys and value types
+    use_mqtt = use_mqtt_protocol(config)
+    sample_protocol_dict = SAMPLE_AMQP_CONFIG if not use_mqtt \
+        else SAMPLE_MQTT_CONFIG
     sample_config = {
-        **SAMPLE_AMQP_CONFIG, **SAMPLE_VCD_CONFIG,
+        **sample_protocol_dict, **SAMPLE_VCD_CONFIG,
```

Comment:
Hi Chintan, I agree that your renaming of "sample_protocol_dict" is more consistent and am changing it. I am leaving it lower cased because "sample_protocol_dict" is a local variable that is either the SAMPLE_AMQP_CONFIG or the SAMPLE_MQTT_CONFIG.